alns/stopping_criteria/MaxRuntime.py

```diff
@@ -1,41 +1,29 @@
 import time
 
+from typing import Optional
+from numpy.random import RandomState
+
 from alns.State import State
 from alns.stopping_criteria.StoppingCriterion import StoppingCriterion
 
 
 class MaxRuntime(StoppingCriterion):
-    def __init__(self, max_runtime: int) -> None:
+    def __init__(self, max_runtime: float):
         """
-        Criterion that stops after a maximum number of iterations.
+        Criterion that stops after a specified maximum runtime.
         """
         if max_runtime < 0:
-            raise ValueError("Max iterations must be non-negative.")
+            raise ValueError("Max runtime must be non-negative.")
 
         self._max_runtime = max_runtime
-        self._elapsed_runtime = None
-        self._start_runtime = None
+        self._start_runtime: Optional[float] = None
```

Comment:
Question: does mypy require a type here?

Reply:
Yea, it would keeping me these errors if I didn't add that:
```
alns/stopping_criteria/MaxRuntime.py:26: error: Incompatible types in assignment (expression has type "float", variable has type "None")
alns/stopping_criteria/MaxRuntime.py:28: error: Unsupported operand types for - ("float" and "None")
```

Reply:
Ah yes! Mypy has its uses, but it can be annoying sometimes. I asked because I normally don't see a lot of typed (private) fields or variables, and figured it had to be to please the tool.